packages/api-v2/src/hooks/useAuthorize.ts

```diff
@@ -6,7 +6,19 @@ import { useAuthContext } from '../AuthProvider';
  */
 const useAuthorize = () => {
     const ctx = useAuthContext();
-    const { data, switchAccount, isLoading, isSuccess, isFetching, isError, refetch, error, loginid } = ctx;
+    const {
+        data,
+        switchAccount,
+        isLoading,
+        isSuccess,
+        isFetching,
+        isError,
+        refetch,
+        error,
+        loginid,
+        isSwitching,
+        isInitializing,
+    } = ctx;
```

Comment:
Would there be an advantage in grouping `isSomething` into some `status` val?

Reply:
Indeed it feels like an AuthContext is starting to handle too many responsibilities ;-/ Might be good topic for any next PRs